example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -99,31 +98,24 @@ bool offb_ctrl_ned(Device &device)
     // NOTE: Use sleep_for() after each velocity-ned command to closely monitor their behaviour.
 
     offboard_log(offb_mode,  " Turn clock-wise 270 deg");
-    for (unsigned i = 0; i < 400; ++i) {
-        device.offboard().set_velocity_ned({0.0f, 0.0f, 0.0f, 270.0f});
-        sleep_for(milliseconds(10));
-    }
+    device.offboard().set_velocity_ned({0.0f, 0.0f, 0.0f, 270.0f});
+    sleep_for(seconds(2));
+
     offboard_log(offb_mode, " Done");
 
     offboard_log(offb_mode, " Go UP 2 m/s, Turn clock-wise 180 deg");
-    for (unsigned i = 0; i < 400; ++i) {
-        device.offboard().set_velocity_ned({0.0f, 0.0f, -2.0f, 180.0f});
-        sleep_for(milliseconds(10));
-    }
+    device.offboard().set_velocity_ned({0.0f, 0.0f, -2.0f, 180.0f});
+    sleep_for(seconds(4));
     offboard_log(offb_mode, " Done...");
 
     offboard_log(offb_mode,  " Turn clock-wise 90 deg");
```

Comment:
Turn clockwise to face East (90 deg.)"

Reply:
It would also be good if at least one of these cases turned anticlockwise to demonstrate that working. Certainly we need that in the test code.

Reply:
Ok, I'll add that.